src/lib/setup.js

```diff
@@ -208,6 +209,18 @@ export default function(app) {
   )
   app.use("/(.well-known/)?apple-app-site-association", siteAssociation)
 
+  /**
```

Comment:
Thanks for the code comment. This feels like the right place stack-wise for this (_after_ those middlewares that will mess this up, but _before_ the actual AB middleware and app routes).